components/utils/filterAlgortithm.js

```diff
@@ -0,0 +1,191 @@
+const nodes = [
```

Comment:
was this supposed to be replaced by GraphSelection...?

Reply:
Was Patric s code. Actually we should not put regular is within components directory